AEPEdge.podspec

```diff
@@ -12,7 +12,7 @@ Pod::Spec.new do |s|
   s.author           = "Adobe Experience Platform SDK Team"
   s.source           = { :git => "https://github.com/adobe/aepsdk-edge-ios.git", :tag => "v#{s.version}-#{s.name}" }
   s.ios.deployment_target = '10.0'
-  s.swift_version = '5.0'
+  s.swift_version = '5.1'
 
   s.pod_target_xcconfig = { 'BUILD_LIBRARY_FOR_DISTRIBUTION' => 'YES' }
   s.dependency 'AEPCore'
```

Comment:
I think s.dependency 'AEPServices' shouldn't be required if it is brought by AEPCore.